src/components/ComponentDocs/ComponentDocs.js

```diff
@@ -9,8 +9,9 @@ export default class ComponentDocs extends React.Component {
   render() {
     const { component } = this.props;
 
-    let componentDocUrl;
-    componentDocUrl = require(`carbon-components/src/components/${component}/README.md`);
+    let componentDocUrl = '';
+    // Loading README.md causes Storybook build failure as of now
+    // componentDocUrl = require(`carbon-components/src/components/${component}/README.md`);
```

Comment:
```suggestion
    componentDocUrl = require(`carbon-components/src/components/${component}/README.md`);
```

For now why don't we revert this and delete the `ComponentDocs.stories.jsx` file. That way it will still work and we can merge this in and I can work on adding the correct stories for each component.

Reply:
Great point - I now see I was too focus on getting the Storybook working!